x/wasm/keeper/keeper_cgo.go

```diff
@@ -63,6 +59,16 @@ func NewKeeper(
 	for _, o := range opts {
 		o.apply(keeper)
 	}
+	// only set the wasmvm if no one set this in the options
+	// NewVM does a lot, so better not to create it and silently drop it.
```

Comment:
Thanks for bringing this up! Unfortunately, the implementation conflicts with the [`WithWasmEngineDecorator`](https://github.com/CosmWasm/wasmd/blob/v0.41.0-rc.0/x/wasm/keeper/options.go#L28) now that requires the object to exist. 
I will come up with some solution